install/policies/deployment.php

```diff
@@ -93,13 +99,15 @@
       'type_data'                           => '',
       'unicity'                             => 0,
       'plugin_flyvemdm_policycategories_id' => $category,
-      'comment'                             => __('Uninstall a file on the device',
+      'comment'                             => __('Uninstall a file on the device. Available since Android 4.2',
```

Comment:
Please drop Available from **** in all policies, as I suggested in my previous intermediate review.